docs/resources/vpn_gateway.md

```diff
@@ -0,0 +1,196 @@
+
+  Changing this parameter will create a new resource.
+
+* `slave_eip` - (Required, String, ForceNew) The slave EIP configurations.
+  The [object](#Gateway_CreateRequestEip) structure is documented below.
+
+  Changing this parameter will create a new resource.
+
+* `attachment_type` - (Optional, String, ForceNew) The attachment type. The value can be **vpc**.
+  Defaults to **vpc**
+
+  Changing this parameter will create a new resource.
+
+* `flavor` - (Optional, String, ForceNew) The flavor of the VPN gateway. The value can be **V1G** and **V300**.
+  Defaults to **V300**
+
+  Changing this parameter will create a new resource.
+
+* `bgp_asn` - (Optional, Int, ForceNew) The ASN number of BGP. The value ranges from **1** to **4294967295**.
+  Defaults to **64512**
+
+  Changing this parameter will create a new resource.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project ID.
+
+  Changing this parameter will create a new resource.
+
+<a name="Gateway_CreateRequestEip"></a>
+The `master_eip` or `slave_eip` block supports:
+
+* `id` - (Optional, String, ForceNew) The public IP ID.
+
+  Changing this parameter will create a new resource.
+
+* `type` - (Optional, String, ForceNew) The EIP type. The value can be **5_bgp** and **5_sbgp**.
+
+  Changing this parameter will create a new resource.
+
+* `bandwidth_name` - (Optional, String, ForceNew) The bandwidth name.
+
+  Changing this parameter will create a new resource.
+
+* `bandwidth_size` - (Optional, Int, ForceNew) Bandwidth size in Mbit/s. When the `flavor` is **V300**, the value
+  cannot be greater than **300**. When the `flavor` is **V1G**, the value cannot be greater than **1024**.
+
+  Changing this parameter will create a new resource.
+
+* `charge_mode` - (Optional, String, ForceNew) The charge mode of the bandwidth. The value can be **bandwidth** and **traffic**.
+
+  Changing this parameter will create a new resource.
+
+  ~> You can use `id` to specify an existing EIP or use `type`, `bandwidth_name`, `bandwidth_size` and `charge_mode` to
+    create a new EIP.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ID of the VPN gateway
+
+* `status` - The status of VPN gateway.
+
+* `created_at` - The create time.
+
+* `updated_at` - The update time.
+
+* `used_connection_group` - The number of used connection groups.
+
+* `used_connection_number` - The number of used connections.
+
+* `master_eip` - The master EIP configurations.
+  The [object](#Gateway_GetResponseEip) structure is documented below.
+
+* `slave_eip` - The slave EIP configurations.
+  The [object](#Gateway_GetResponseEip) structure is documented below.
+
+<a name="Gateway_GetResponseEip"></a>
+The `master_eip` or `slave_eip` block supports:
+
+* `bandwidth_billing_info` - The bandwidth billing info.
+
+* `bandwidth_id` - The bandwidth ID.
+
+* `billing_info` - The public IP billing info.
```

Comment:
what are the value of `billing_info` and `bandwidth_billing_info`? I think they are meaningless

Reply:
they removed